tools/breaking-change-detector/GNUmakefile

```diff
@@ -0,0 +1,16 @@
+localsetup:
```

Comment:
Is there any documentation on how to use this? What does someone have to call localsetup with?